sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -318,5 +368,17 @@ public virtual async Task<Response<AccessToken>> GetTokenForTeamsUserAsync(GetTo
                 throw;
             }
         }
+
+        private static int? GetTokenExpirationInMinutesOrNull(TimeSpan tokenExpiresAfter)
+        {
+            try
+            {
+                return tokenExpiresAfter == default ? null : Convert.ToInt32(tokenExpiresAfter.TotalMinutes);
+            }
+            catch (OverflowException ex)
+            {
+                throw new ArgumentException($"The {nameof(tokenExpiresAfter)} argument is out of permitted bounds. Please refer to the documentation and set the value accordingly.", ex);
```

Comment:
This is technically not correct. Please grab the name of the parameter from the calling method using nameof...as in the example I sent you.

Reply:
Done as agreed.